test/integration/targets/pip/tasks/pip.yml

```diff
@@ -496,7 +496,10 @@
 
   - name: install distribute in the virtualenv
     pip:
-      name: distribute
+      name:
+        - distribute
+        # setuptools 45.0.0 no longer works with Python 2
+        - setuptools<45.0.0
```

Comment:
```suggestion
 - setuptools<45; python_version < "3.0"
```

Reply:
The task is contained in a block with `when: ansible_python.version.major == 2`.